src/pages/javascript/tutorials/using-anonymous-functions-for-private-namespacing-in-your-apps/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Using Anonymous Functions for Private Namespacing in Your JavaScript Apps
+title: Using Anonymous Functions for Private Namespacing in Your Apps
```

Comment:
I'm sorry for requesting a change to something that was already like this the last time I reviewed. Can you change this to "Using Anonymous Functions for Private Namespacing"?